lib/src/lints/avoid_debug_print_in_release/avoid_debug_print_in_release_rule.dart

```diff
@@ -31,23 +32,23 @@ import 'package:solid_lints/src/models/solid_lint_rule.dart';
 /// ```
 ///
 ///
-class AvoidDebugPrint extends SolidLintRule {
+class AvoidDebugPrintInReleaseRule extends SolidLintRule {
   /// The [LintCode] of this lint rule that represents
   /// the error when debugPrint is called
-  static const lintName = 'avoid_debug_print';
+  static const lintName = 'avoid_debug_print_in_release';
 
-  AvoidDebugPrint._(super.config);
+  AvoidDebugPrintInReleaseRule._(super.config);
 
-  /// Creates a new instance of [AvoidDebugPrint]
+  /// Creates a new instance of [AvoidDebugPrintInReleaseRule]
   /// based on the lint configuration.
-  factory AvoidDebugPrint.createRule(CustomLintConfigs configs) {
+  factory AvoidDebugPrintInReleaseRule.createRule(CustomLintConfigs configs) {
     final rule = RuleConfig(
       configs: configs,
       name: lintName,
       problemMessage: (_) => "Avoid using 'debugPrint'",
```

Comment:
We need to update the problem message to let the user know what the real problem is.